test/ctia/bundle/routes_test.clj

```diff
@@ -505,7 +533,21 @@
 
        (is (= 2 (count (:sightings bundle-get-res-4))))
        (is (nil? (:indicators bundle-get-res-4)))
-       (is (= 402 (count (:relationships bundle-get-res-4))))))))
+       (is (= 402 (count (:relationships bundle-get-res-4))))
+
+       (is (= bundle-get-res-3 bundle-get-res-5))
+
+       (is (= indicator-id (-> bundle-from-source
+                               :indicators
+                               first
+                               :id)))
+       (is (= sighting-id-2 (-> bundle-from-source
+                               :sightings
+                               first
+                               :id)))
```

Comment:
Could you check that only the expected entities are returned? The test becomes very difficult to read, could you add `testing` blocks or a message associated with some `is` assertions?